### PR TITLE
improvement: python respects ir50, inserts defaults

### PR DESCRIPTION
**Pull request by armandobelardo**
This adds in long and bool defaults into the openapi parsing, as well as has python starting to input defaults to the model

### CODE REVIEW

File: seed/python-sdk/validation/with-defaults/src/seed/client.py [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+        except JSONDecodeError:
+            raise ApiError(status_code=_response.status_code, body=_response.text)
+        raise ApiError(status_code=_response.status_code, body=_response_json)
+
+    def get(
+        self,
+        *,
+        decimal: float = 1,
+        even: int = 42,
+        name: str = "fern",
```
Comment (dsinghvi) [inline seed/python-sdk/validation/with-defaults/src/seed/client.py:121]: let's say the default for a field typed as optional<integer> is 4, can the user still send `None` ? Is it worth adding an explicit test for that in the test definition

Comment (armandobelardo) [reply seed/python-sdk/validation/with-defaults/src/seed/client.py:121]: I'll add a test
But yeah pydantic will prefer user provided over default

File: packages/cli/openapi-parser/src/schema/convertNumber.ts [hunk 2]
```diff
@@ -80,6 +80,7 @@ export function convertNumber({
             nameOverride,
             generatedName,
             primitive: PrimitiveSchemaValueWithExample.int64({
+                default: _default,
```
Comment (dsinghvi) [inline packages/cli/openapi-parser/src/schema/convertNumber.ts:83]: would be great to add a fixture (or update existing) to check that the parsing for longs and booleans work?

**Review by dsinghvi — APPROVED** [review]
looks great would be sick if we were able to include the `object_with_defaults.py` test in the generated SDK (if the customer had objects with deafults)

but ik this is asking a lot

**Pull request merged**